MetadataMigration/src/main/java/org/opensearch/migrations/clusters/SourceCluster.java

```diff
@@ -0,0 +1,12 @@
+package org.opensearch.migrations.clusters;
```

Comment:
Classes such as `SourceCluster` should be in the centralized project rather than metadata specific.